docs/content/2.deploy/20.providers/cloudflare.md

```diff
@@ -10,17 +10,15 @@ Deploy Nitro apps to CloudFlare.
 **Note:** This preset uses [service-worker syntax](https://developers.cloudflare.com/workers/learning/service-worker/) for deployment.
 ::
 
-Login to your [Cloudflare Workers](https://workers.cloudflare.com) account and obtain your `account_id` from the sidebar.
-
 Create a `wrangler.toml` in your root directory:
 
 ```ini
 name = "playground"
 main = "./.output/server/index.mjs"
 workers_dev = true
-compatibility_date = "2022-09-10"
-account_id = "<the account_id you obtained (optional)>"
-route = "<mainly useful when you want to setup custom domains (optional too)>"
+compatibility_date = "2023-12-01"
+account_id = "<(optional) your Cloudflare account id, retrievable from the Cloudflare dashboard>"
+route = "<(optional) mainly useful when you want to setup custom domains>"
```

Comment:
Shall we comment them by default? 

```suggestion
# route = "<(optional) mainly useful when you want to setup custom domains>"
```

Reply:
yeah sounds good to me so that it doesn't generate errors if people just copy-paste the toml 🙂